cmd/main.go

```diff
@@ -129,14 +129,29 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = (&controllers.PDBWatcherReconciler{
+	if err = (&controllers.PodScalingReconciler{
```

Comment:
nitpick totally get this but the thing being "reconciled" is the pdbwaters (thats the object bein watched and then two objects are ipdated it and deployment/statefulset)

You could call it the pod scaler but podscaling reconciler is wierd.

Reply:
https://chatgpt.com/share/674b3fff-0500-8009-be05-29a20329fdc4